src/System.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -158,21 +152,15 @@ protected virtual HashAlgorithmName GetHashAlgorithmName(string algorithm)
 
             switch (algorithm)
             {
-                case SecurityAlgorithms.SHA256:
```

Comment:
I removed SecurityAlgorithms.SHA256 (and the equivalents) because it's not really a signature algorithm, but I'm fine reverting this change if you think it's better to keep it.